.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @andrewgy8 @tobami @edgarlatorre @daniel-ruiz
+* @andrewgy8 @tobami @edgarlatorre @daniel-ruiz @EmilioCarrion @mgarcia0094 @AdamjGoddard
```

Comment:
```suggestion
* @daniel-ruiz @EmilioCarrion @mgarcia0094 @AdamjGoddard
```
😜

Reply:
<img src="https://c.tenor.com/O2RBK9klEMYAAAAC/homer-simpson-homer.gif" width="200" height="200" />

Reply:
I forgot to upload the last commit D:. Thank you!